src/bun.js/readline.exports.js

```diff
@@ -0,0 +1,3122 @@
+Interface.prototype._deleteWordLeft = _Interface.prototype[kDeleteWordLeft];
+Interface.prototype._deleteWordRight = _Interface.prototype[kDeleteWordRight];
+Interface.prototype._deleteLineLeft = _Interface.prototype[kDeleteLineLeft];
+Interface.prototype._deleteLineRight = _Interface.prototype[kDeleteLineRight];
+Interface.prototype._line = _Interface.prototype[kLine];
+Interface.prototype._historyNext = _Interface.prototype[kHistoryNext];
+Interface.prototype._historyPrev = _Interface.prototype[kHistoryPrev];
+Interface.prototype._getDisplayPos = _Interface.prototype[kGetDisplayPos];
+Interface.prototype._getCursorPos = _Interface.prototype.getCursorPos;
+Interface.prototype._moveCursor = _Interface.prototype[kMoveCursor];
+Interface.prototype._ttyWrite = _Interface.prototype[kTtyWrite];
+
+function _ttyWriteDumb(s, key) {
+  key = key || kEmptyObject;
+
+  if (key.name === "escape") return;
+
+  if (this[kSawReturnAt] && key.name !== "enter") this[kSawReturnAt] = 0;
+
+  if (keyCtrl) {
+    if (key.name === "c") {
+      if (this.listenerCount("SIGINT") > 0) {
+        this.emit("SIGINT");
+      } else {
+        // This readline instance is finished
+        this.close();
+      }
+
+      return;
+    } else if (key.name === "d") {
+      this.close();
+      return;
+    }
+  }
+
+  switch (key.name) {
+    case "return": // Carriage return, i.e. \r
+      this[kSawReturnAt] = DateNow();
+      this._line();
+      break;
+
+    case "enter":
+      // When key interval > crlfDelay
+      if (
+        this[kSawReturnAt] === 0 ||
+        DateNow() - this[kSawReturnAt] > this.crlfDelay
+      ) {
+        this._line();
+      }
+      this[kSawReturnAt] = 0;
+      break;
+
+    default:
+      if (typeof s === "string" && s) {
+        this.line += s;
+        this.cursor += s.length;
+        this._writeToOutput(s);
+      }
+  }
+}
+
+// ----------------------------------------------------------------------------
+// Exports
+// ----------------------------------------------------------------------------
+export var Interface = Interface;
+export var clearLine = clearLine;
+export var clearScreenDown = clearScreenDown;
+export var createInterface = createInterface;
+export var cursorTo = cursorTo;
+export var emitKeypressEvents = emitKeypressEvents;
+export var moveCursor = moveCursor;
+export var promises = {
+  [SymbolFor("__UNIMPLEMENTED__")]: true,
+};
+
+export default {
+  Interface,
+  clearLine,
+  clearScreenDown,
+  createInterface,
+  cursorTo,
+  emitKeypressEvents,
+  moveCursor,
+  promises,
+
+  [SymbolFor("__BUN_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED__")]: {
+    CSI,
+    _Interface,
+    utils: {
+      getStringWidth,
+      stripVTControlCharacters,
+    },
+    shared: {
+      kEmptyObject,
+      validateBoolean,
+      validateInteger,
+      validateAbortSignal,
+      ERR_INVALID_ARG_TYPE,
+    },
+    symbols: {
+      kQuestion,
+      kQuestionCancel,
+    },
+  },
+  [SymbolFor("CommonJS")]: true,
```

Comment:
This needs to be `0` when `default` is not a function, see https://github.com/oven-sh/bun/blob/743c66cccf045e9a1c4960693e8af3cb54263501/src/bun.js/builtins/js/ImportMetaObject.js#L150-L156

Reply:
Ah ok I missed that